src/Uno.UI/Controls/CommandBarRenderer.Android.cs

```diff
@@ -123,6 +123,9 @@ protected override IEnumerable<IDisposable> Initialize()
 				new[] { CommandBar.VisibilityProperty },
 				new[] { CommandBar.PaddingProperty },
 				new[] { CommandBar.OpacityProperty },
+				new[] { CommandBar.HorizontalContentAlignmentProperty },
```

Comment:
whaouh .... that's lot of arrays with often only on item :/

Reply:
Yes, but it's either that, or a big switch case ? Or react on any property change.